pyproject.toml

```diff
@@ -11,9 +11,9 @@ packages = [
   
 
 [tool.poetry.dependencies]
-python = "^3.6.2,<3.10"
+python = "^3.10"
 numpy = "^1.19.0"
-casadi = "^3.5.4,!=3.5.5.post1,!=3.5.5.post2"
+casadi = "3.5.6.pre2"
```

Comment:
This casadi library is preciesly the issue with python 3.10. Sometimes it cannot be found directly.

Reply:
it has been updated to "3.5.6.pre2, ^3.5.5,"